app/src/main/res/values/colors.xml

```diff
@@ -6,6 +6,7 @@
     <color name="colorAccentExposure">#A5E33C68</color>
     <dimen name="exposure_summary_text">20sp</dimen>
     <dimen name="padding_content">16dp</dimen>
+    <dimen name="small_padding_content">8dp</dimen>
```

Comment:
I did declare bunch of padding values in `dimens.xml`. And I'm not sure if we should declare them in `colors.xml` :)

Reply:
My bad...
I use AS default dialog for creation of undeclared resources and I hoped it put them into right files. But it's not! :man_shrugging: 

Thanks for catching this : )